### PR TITLE
Feature/abrfc01 cnrfc06 qpe

### PR DESCRIPTION
**Pull request by jeffsuperglide**
Added ABRFC and CNRFC together.  This branch supersedes `abrfc-qpe-01h` and `cnrfc-qpe-06h` branches.

### CODE REVIEW

File: sql/common/V2.07.00__cnrfc_qpe.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+INSERT INTO acquirable (id, name, slug) VALUES
+    ('34a89c35-090d-46e8-964a-c621403301b9', 'cnrfc-qpe-06h', 'cnrfc-qpe-06h');
+
+-- add product suite
+INSERT INTO suite (id, name, slug, description) VALUES
+    ('469552d5-51f8-40d3-b1f4-658af894c3e8', 'California Nevada River Forecast Center (CNRFC)', 'cnrfc', 'CNRFC Description');
+
+-- add product
+INSERT INTO product (id,slug,"label",temporal_duration,temporal_resolution,dss_fpart,parameter_id,description,unit_id,deleted,suite_id) VALUES
+    ('eac3976f-6503-442d-bf20-1a2ffe555162','cnrfc-qpe','QPE',21600,21600,'CNRFC-QPE','eb82d661-afe6-436a-b0df-2ab0b478a1af','CNRFC Quantitative Precipitation Estimate (QPE)','e245d39f-3209-4e58-bfb7-4eae94b3f8dd',false,'469552d5-51f8-40d3-b1f4-658af894c3e8');
```
Comment (brettpalmberg) [inline sql/common/V2.07.00__cnrfc_qpe.sql]: Recommend changing slug to `cnrfc-qpe-06h`

**Review by brettpalmberg — CHANGES_REQUESTED** [review]

File: sql/common/V2.07.00__cnrfc_qpe.sql (hunk 1 above)
Comment (jeffsuperglide) [reply sql/common/V2.07.00__cnrfc_qpe.sql]: Updated.  Did it locally when testing but change didn't make it through the commit.

**Review by brettpalmberg — APPROVED** [review]

File: sql/common/V2.07.00__cnrfc_qpe.sql (hunk 1 above)
Comment (brettpalmberg) [reply sql/common/V2.07.00__cnrfc_qpe.sql]: 👍🏻 Looks good @jeffsuperglide. Let's have @adamscarberry take a look too when convenient, he has done more on naming/labeling products than I have at this point :) After that, feel free to merge to `develop`.

Comment (adamscarberry) [reply sql/common/V2.07.00__cnrfc_qpe.sql]: oops, good catch

**Review by adamscarberry — APPROVED** [review]

**Comment by adamscarberry** [comment]
https://github.com/USACE/cumulus/issues/248

https://github.com/USACE/cumulus/issues/253

**Pull request merged**